examples/data-objects/task-selection/src/app.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { getTinyliciousContainer } from "@fluid-experimental/get-container";
+import { IContainer } from "@fluidframework/container-definitions";
+
+import { oldestClientDiceId, taskManagerDiceId, TaskSelectionFactory } from "./containerCode";
+import { IDiceRoller } from "./interface";
+import { renderDiceRoller } from "./view";
+
+// In interacting with the service, we need to be explicit about whether we're creating a new document vs. loading
+// an existing one.  We also need to provide the unique ID for the document we are creating or loading from.
+
+// In this app, we'll choose to create a new document when navigating directly to http://localhost:8080.  For the ID,
+// we'll choose to use the current timestamp.  We'll also choose to interpret the URL hash as an existing document's
+// ID to load from, so the URL for a document load will look something like http://localhost:8080/#1596520748752.
+// These policy choices are arbitrary for demo purposes, and can be changed however you'd like.
+let createNew = false;
+if (location.hash.length === 0) {
+    createNew = true;
+    location.hash = Date.now().toString();
+}
+const documentId = location.hash.substring(1);
+document.title = documentId;
+
+async function requestDiceRoller(container: IContainer, id: string): Promise<IDiceRoller> {
+    const response = await container.request({ url: id });
```

Comment:
Boiler-plate code can be replaced by using requestFluidObject<IDiceRoller>() instead.

Reply:
Actually, whole function is doing what can be done by a call to requestFluidObject

Reply:
Good call, will replace.